src/ElasticSan/ElasticSan.Autorest/docs/New-AzElasticSan.md

```diff
@@ -14,18 +14,18 @@ create ElasticSan.
 
 ### CreateExpanded (Default)
 ```
-New-AzElasticSan -Name <String> -ResourceGroupName <String> -BaseSizeTiB <Int64>
- -ExtendedCapacitySizeTiB <Int64> -Location <String> -SkuName <String> [-SubscriptionId <String>]
- [-AvailabilityZone <String[]>] [-PublicNetworkAccess <String>] [-SkuTier <String>] [-Tag <Hashtable>]
+New-AzElasticSan -Name <String> -ResourceGroupName <String> -Location <String> -SkuName <String>
```

Comment:
It would be better to update help description of the 2 parameters to add the default value of them. 
You might can check with Xiaogang to see if any easy way to do that.